nservicebus/recoverability/configure-delayed-retries.md

```diff
@@ -17,7 +17,7 @@ related:
 
 NOTE: Starting from NServiceBus Version 6 Delayed Retries Policy (formerly known as Second Level Retries Policy) has been deprecated in favor of the new custom Recoverability policy which allows much more control over the Recoverability behavior. This documentation shows how previous Delayed Retries Policies can be implemented with the new [Recoverability Policy](/nservicebus/recoverability/custom-recoverability-policy.md).
 
-WARNING: Delayed Retries cannot be used when transport transactions are disabled or Delayed Delivery is not available. For more information about transport transactions, refer to [transport transaction](/nservicebus/transports/transactions.md). Details regarding delayed delivery can be found in [Delayed Delivery](/nservicebus/messaging/delayed-delivery.md#caveats) article.
+WARNING: Delayed Retries cannot be used when transport transactions are disabled or Delayed Delivery is not available. For more information about transport transactions, refer to [transport transaction](/nservicebus/transports/transactions.md). For more details on the caveats, see: [Delayed Delivery and its Caveats](/nservicebus/messaging/delayed-delivery.md#caveats) article.
```

Comment:
Isn't this unnecessary repetition? Now we have 

For more details on the _caveats_, see: [Delayed Delivery and its _Caveats_](/nservicebus/messaging/delayed-delivery.md#caveats) article

see italic _caveats_

How about removing the first one? and just write 

For more details, see the [Delayed Delivery and its Caveats](/nservicebus/messaging/delayed-delivery.md#caveats) article.